RELEASE_NOTES.md

```diff
@@ -0,0 +1,147 @@
+# BEAM Release Notes
+## 1.0.0
+### New Features
+- DemandFollowingRepositionig: Reposition the random activity's location.
+- Added CSV and XML writer for PhysSim events.
+- Generate secondary activities.
+- R5: Limit routing by max distance for BIKE.
+- Inverse square demand following repositioning.
```

Comment:
maybe we can put things related to ridehail/repositioning in one place and give it a heading - not required, just a thought, if we can identify groups of features

Reply:
same for router and maybe mode choice

Reply:
or parking

Reply:
freight?

Reply:
you can create additional headings as needed to group them